### ISSUE TITLE
(OUI Docs) Page Header

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/oui#178]
- [ ] Remove logo from "Page title". 

Note: This logo did not previously appear, I believe a bug fix may have revealed it. However, we do not use logos or icons in our page titles.

**Comment by KrooshalUX** [comment]
Blocked by https://github.com/opensearch-project/oui/issues/113

**Comment by KrooshalUX** [comment]
closed with #625

**Comment by KrooshalUX** [comment]
This appears closed, and the associated PR is merged and closed, but I still see a kibana icon in both 1.0 and 1.1 versions

![Uploading Screen Shot 2023-05-02 at 12.39.31 PM.png…]()

**Comment by anirudha** [comment]
Closing — this documentation issue has been open since 2022/2023 with no progress. The doc site has been significantly updated since then. If specific documentation gaps remain, please open a new issue with current context.
